vmdb/db/migrate/20100308005125_create_miq_cim_instances.rb

```diff
@@ -14,7 +14,7 @@ def self.up
       t.column :agent_id,         :integer
       t.column :stat_id,          :integer
       t.column :stat_top_id,        :integer
-      t.timestamps
+      t.timestamps :null => true
```

Comment:
Well, hopefully people are running them locally and will get the warning but yes, when rails5 comes and defaults to false and if the warning goes away, yes, you'll have no idea.